tests/contrib/pylons/test_pylons.py

```diff
@@ -690,6 +716,11 @@ def test_pylons_body_xml_attack(self):
 
             root_span = spans[0]
             assert root_span
+
+            assert (
+                "UnicodeDecodeError: 'utf8' codec can't decode byte 0x80 in position 0: invalid start byte"
+                not in self._caplog.text
```

Comment:
```suggestion
                "UnicodeDecodeError" not in self._caplog.text
```

same here too, optional, just suggesting to make it easier to update both if you decided to.